### ISSUE TITLE
Error having to do with the filename of boxplot-methods.R

### ISSUE THREAD

**Issue by jwbowers** [markmfredrickson/optmatch#99]
I had trouble building and installing the latest version of optmatch (master branch).  Here is some of the text surrounding the error:

```
g++ -shared -L/usr/local/lib64 -o optmatch.so cuseful.o distances.o ism.o map.o r_smahal.o register.o relax4s.o relax4sold.o smahal.o subsetInfSparseMatrix.o -L/usr/local/intel/intel-15.0/mkl/lib/intel64 -Wl,--start-group -lmkl_gf_lp64 -lmkl_sequential -lmkl_core -Wl,--end-group -lgfortran -lm -lgfortran -lm
make[1]: Leaving directory `/tmp/RtmpRbTNQ6/R.INSTALL3fbe3c611a71/optmatch/src'
installing to /home/jwbowers/Documents/PROJECTS/Reg-No-Regrets/libraries/optmatch/libs
** R
Error in .install_package_code_files(".", instdir) :
files in 'Collate' field missing from '/tmp/RtmpRbTNQ6/R.INSTALL3fbe3c611a71/optmatch/R':
  boxplot- methods.R
ERROR: unable to collate and parse R files for package 'optmatch'
* removing '/home/jwbowers/Documents/PROJECTS/Reg-No-Regrets/libraries/optmatch'
make: *** [libraries/optmatch/INSTALLED] Error 1
```

I discovered that I was not the only person having this problem: <https://stackoverflow.com/questions/14728326/files-in-collate-field-missing-from-package-when-installing-from-github> 

As a very quick hack I just did:

```
mv boxplot-methods.R boxplotMethods.R
```

And now it builds even though it complains about methods for boxplot.

I have not committed this change because it is very easy to implement by others and perhaps my 1 second of thought about file names is not the correct answer to this problem.

**Comment by benthestatistician** [comment]
I can reproduce the problem (complaints about not finding boxplot methods after the file name change), but I can't see why it happens. After the rename I'm seeing the problem after running `make tests` locally, ie without `install_github` (which seems to be a part of the issue in the linked thread).  

Renaming to `boxplotmethods.R` also doesn't fix.  greping in optmatch and subdirectories didn't identify a reference to "boxplot-methods.R".  Hmm.  Hints, anyone?

**Comment by benthestatistician** [comment]
Fixed now.  Main clue: In my optmatch/ directory I find a file called `.pkgexclude`, among other things containing the line `boxplotMethods.R`. Modifying it directly doesn't help, because the Makefile creates it on the fly, per the line
````
git ls-files --other --exclude-standard > .pkgexclude
```
In other words, the problem was that I hadn't yet staged or committed the change of filenames. I also had to run a `make clean` to conclusively ditch the problem.   Implemented in master 1ff6734.  Closing.
